lib/module.js

```diff
@@ -21,6 +21,8 @@ module.exports = function sentry (moduleOptions) {
     publishRelease: process.env.SENTRY_PUBLISH_RELEASE || false,
     disableServerRelease: process.env.SENTRY_DISABLE_SERVER_RELEASE || false,
     disableClientRelease: process.env.SENTRY_DISABLE_CLIENT_RELEASE || false,
+    attachCommits: process.env.SENTRY_AUTO_ATTACH_COMMITS !== '0',
```

Comment:
Sorry if I wasn't clear but I think `false` also needs to be treated as falsey value. I meant to handle 0 as an extra.
Still would suggest extra helper function.

Reply:
Still not addressed but was resolved.